### PR TITLE
Feature/per channel broadcast

### PR DESCRIPTION
**Pull request by Kiryous**

### CODE REVIEW

File: src/dapp-message/dapp-message.controller.dto.ts [hunk 1]
```diff
@@ -4,12 +4,19 @@ import {
...
   IsString,
   IsUUID,
+  ValidateIf,
 } from 'class-validator';
-import { AddressTypeDto } from 'src/address/address.controller.dto';
+import { AddressTypeDto } from '../address/address.controller.dto';
 import { IsPublicKey } from '../middleware/public-key-validation';
 
+// Title is mandatory only for EMAIL only broadcasts or when is addressTypes is not specified
+const isTitleNeeded = (obj: any) =>
```
Comment (tsmbl) [inline src/dapp-message/dapp-message.controller.dto.ts:13]: Why `any`?

**Review by tsmbl — APPROVED** [review]
Looks cool, also thanks for adding validator! Left some comments related to typing

**Review by tsmbl — CHANGES_REQUESTED** [review]
Let's fix typing comments and the gtg
Do you also plan to update sdk?

**Review by tsmbl — CHANGES_REQUESTED** [review]
Hey, thanks for addressing comments!

I've noticed a couple of compilation errors after your latest changes, could you please fix them?

**Comment by Kiryous** [comment]
@tsmbl thanks for reviewing, esp on weekend! sorry, forgot to push something. Now ready!

**Review by tsmbl — APPROVED** [review]
Lgtm

**Pull request merged**